src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search-bar search-bar--big"
```

Comment:
The attributes of the `<form>` element should be formatted correctly. Each attribute should start on a new line with a 2-space indent relative to the tag.